model/vm.rb

```diff
@@ -51,6 +51,7 @@ def display_state
   end
 
   def mem_gib_ratio
+    return 3 if arch == "arm64"
```

Comment:
Should this be 3.2 and we use rounding to make the gigabyte hugepages work out? This does result in some odd stuff like certain plan sizes are more efficient than others...but, why not? If you buy five cores, why not 16GB?

Reply:
You are fast ⚡ I was expecting this comment. I changed it.